client/app/i18n/locales/de/Dashboards.json

```diff
@@ -0,0 +1,83 @@
+{
+  "Choose Visualization": "Visualisierung wählen",
+  "Widget could not be added": "Widget konnte nicht hinzugefügt werden",
+  "Add Widget": "Widget hinzufügen",
+  "Add to Dashboard": "Zum Dashboard hinzufügen",
+  "Parameters": "Parameter",
+  "New Dashboard": "Neues Dashboard",
+  "Dashboard Name": "Dashboard-Name",
+  "Dashboard name": "Dashboard-Name",
+  "This widget requires access to a data source you don&apos;t have access to.": "Dieses Widget erfordert Zugriff auf eine Datenquelle, auf die Sie keinen Zugriff haben.",
+  "Download as CSV File": "Als CSV-Datei herunterladen",
+  "Download as TSV File": "Als TSV-Datei herunterladen",
+  "Download as Excel File": "Als Excel-Datei herunterladen",
+  "View Query": "Abfrage anzeigen",
+  "Edit_Parameters": "Parameter bearbeiten",
+  "Refreshing...": "Wird aktualisiert...",
+  "Refreshing, please wait. ": "Wird aktualisiert, bitte warten.",
+  "Press to refresh. ": "Drücken Sie zum Aktualisieren.",
```

Comment:
Maybe better "Zum Aktualisieren drücken"

Reply:
Yep, that's better

Reply:
thanks, please find it updated here https://github.com/metr-systems/redash/pull/15/commits/08f76697c41782975e8535194d67bbbea43039aa